src/khoj/routers/api.py

```diff
@@ -740,3 +724,61 @@ async def extract_references_and_questions(
         compiled_references = [item.additional["compiled"] for item in result_list]
 
     return compiled_references, inferred_queries, defiltered_query
+
+
+# Stripe integration for Khoj Cloud Subscription
+stripe.api_key = os.getenv("STRIPE_API_KEY")
+endpoint_secret = os.getenv("STRIPE_SIGINING_SECRET")
```

Comment:
Similar to `auth.py`, we can move this into a separate route file, and add a warning for users if they don't have these environment variables set.

And if they are not set, we probably want to return 404s for any of the below endpoints?

Reply:
Moved the subscription routes to separate route file and disabled the API endpoints when stripe environment not setup. 

Showing when billing is enabled in logs instead of warning as default self-hosted users can't do anything about it (unlike the Google OAuth warning where `--anonymous-mode` should be turned on so warning is a good idea)